common/CMakeLists.txt

```diff
@@ -138,3 +138,15 @@ target_include_directories(identity_parser_test PRIVATE ${CMAKE_SOURCE_DIR})
 gtest_discover_tests(identity_parser_test)
 add_dependencies(check identity_parser_test)
 
+
+find_package(Boost 1.54 REQUIRED COMPONENTS filesystem)
+add_library(common_inventory_parser STATIC inventory_parser/inventory_parser.cpp)
+target_link_libraries(common_inventory_parser PUBLIC crossplatform_compiler_flags)
+target_include_directories(common_inventory_parser PRIVATE ../ ${CMAKE_BINARY_DIR} ${Boost_INCLUDEDIR})
```

Comment:
Make sure you coordinate the `../` cleanup with your other effort.

Reply:
This will be taken care of in #1154, I want to merge this one first.